Fix flags passed to `rustc`.

utkarshkukreti opened the pull request:
The flag `--no-trans` was replaced by `-Z no-trans` recently: https://github.com/rust-lang/rust/commit/953f294

lunaryorn commented:
@utkarshkukreti Is that change part of any Rust release already?  If so, please update the docstring accordingly to state the new requirement, otherwise I'd like to wait for the next rust release before merging this.

utkarshkukreti commented:
Just checked that it is available in the latest release, [1.0.0 alpha](http://blog.rust-lang.org/2015/01/09/Rust-1.0-alpha.html).
I have updated the comment and rebased.

lunaryorn commented:
@utkarshkukreti Thanks, merged.
